### PR TITLE
[SeoBundle] Fixed issue that OG image can not be deleted.

### PR DESCRIPTION
**Pull request by bureaublauwgeel**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | 428

### CODE REVIEW

**Comment by roderik** [comment]
@krispypen :+1:

File: src/Kunstmaan/SeoBundle/Entity/Seo.php [hunk 1]
```diff
@@ -350,7 +350,7 @@ public function getOgDescription()
      *
      * @return Seo
      */
-    public function setOgImage(Media $ogImage)
```
Comment (krispypen) [inline src/Kunstmaan/SeoBundle/Entity/Seo.php]: Removing the type is not the fix for a null-exception. Can you change this to "Media $ogImage = null" ?

Comment (bureaublauwgeel) [inline src/Kunstmaan/SeoBundle/Entity/Seo.php]: Should be changed.

**Comment by wimvds** [comment]
Replaced with https://github.com/Kunstmaan/KunstmaanBundlesCMS/pull/452

**Pull request closed without merging**